docs/tutorials/aws/semantic_search_with_cohere_embedding_model.md

```diff
@@ -0,0 +1,371 @@
+# Topic
+
+> The easiest way for setting up embedding model on your Amazon OpenSearch cluster is using [AWS CloudFormation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/cfn-template.html)
+
+> This tutorial explains detail steps if you want to configure everything manually. You can also connect to other service with similar way.
+
+This doc introduces how to build semantic search in Amazon managed OpenSearch with [Cohere embedding model](https://docs.cohere.com/reference/embed).
+If you are not using Amazon OpenSearch, you can refer to [cohere_v3_connector_embedding_blueprint](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/remote_inference_blueprints/cohere_v3_connector_embedding_blueprint.md) and [OpenSearch semantic search](https://opensearch.org/docs/latest/search-plugins/semantic-search/).
+
+
+Note: You should replace the placeholders with prefix `your_` with your own value
+
+# Steps
+
+## 0. Create OpenSearch cluster
+
+Go to AWS OpenSearch console UI and create OpenSearch domain.
+
+Copy the domain ARN which will be used in later steps.
+
+## 1. Create secret
```

Comment:
Should we mention in the Topics that this cohere model is available in bedrock so we can basically use the same steps for the bedrock service to run semantic search using the cohere model?

Reply:
sure